modules/wowza/main.tf

```
@@ -7,51 +7,65 @@ resource "azurerm_resource_group" "rg" {
   location = var.location
   tags     = var.common_tags
 }
+locals {
+  main_container_name = "recordings"
+}
+module "sa" {
+  source = "git::https://github.com/hmcts/cnp-module-storage-account.git?ref=master"
+  #source = "../../../cnp-module-storage-account"
+
+  env = var.env
+
+  storage_account_name = "${replace(lower(local.service_name), "-", "")}sa"
+  common_tags          = var.common_tags
+
+  resource_group_name = azurerm_resource_group.rg.name
+  location            = azurerm_resource_group.rg.location
 
-resource "azurerm_storage_account" "sa" {
-  name                      = "${replace(lower(local.service_name), "-", "")}sa"
-  resource_group_name       = azurerm_resource_group.rg.name
-  location                  = azurerm_resource_group.rg.location
-  tags                      = var.common_tags
-  access_tier               = var.sa_access_tier
-  account_kind              = var.sa_account_kind
-  account_tier              = var.sa_account_tier
-  account_replication_type  = var.sa_account_replication_type
-  enable_https_traffic_only = true
-  min_tls_version           = "TLS1_2"
-  blob_properties {
-    delete_retention_policy {
-      days = 365
+  account_tier             = var.sa_account_tier
+  account_kind             = var.sa_account_kind
+  account_replication_type = var.sa_account_replication_type
+  access_tier              = var.sa_access_tier
+
+  team_name    = "CVP DevOps"
+  team_contact = "#vh-devops"
+
+  policy = [
+    {
+      name = "RecordingRetionPolicy"
```

is this mis-spelled because of character limits?
could shorten it to:
```suggestion
      name = "Recording"
```
or

```suggestion
      name = "RecordingRetention"
```